src/item/components/ItemVideoDescription.scss

```diff
@@ -14,6 +14,16 @@
 		}
 	}
 
+	.c-toolbar__left {
+		flex-grow: 1;
+		max-width: 100%;
+		padding-left: 0.8rem;
```

Comment:
Why is this padding added? It misaligns the title

Reply:
dat moest padding-right zijn
om wat plaats te geven tussen de titel en de bookmark button
![image](https://user-images.githubusercontent.com/1710840/195651034-617a088d-4c9f-418c-8266-a5a9d0cc0e92.png)